traffic_ops/testing/api/v14/cdnfederations_test.go

```diff
@@ -107,6 +116,111 @@ func GetTestCDNFederations(t *testing.T) {
 	}
 }
 
+func AssignTestFederationFederationResolvers(t *testing.T) {
+	// Setup
+	if len(fedIDs) < 1 {
+		t.Fatal("no federations to test")
+	}
+
+	frCnt := len(testData.FederationResolvers)
+	if frCnt < 1 {
```

Comment:
You're still referencing `frIDs[1]` so this should make sure there are at least _two_ Federation Resolvers in the test data.